docs/processes/qualityassurance/templates/bug-report-template.md

```diff
@@ -0,0 +1,27 @@
+## **Bug Report Template**
```

Comment:
**suggestion:** Headers are already with 'bold' font. I think we can drop the `**`. WDYT?